tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh

```diff
@@ -421,7 +430,36 @@ install_cc_tdx_kernel() {
 
 install_cc_sev_kernel() {
 	kernel_version="$(yq r $versions_yaml assets.kernel.sev.version)"
-	install_cc_tee_kernel "sev" "${kernel_version}"
+	
+	export kernel_version=${kernel_version}
+	export KATA_BUILD_CC=yes
+	config_version=$(get_config_version)
+	module_dir="${builddir}/kata-linux-${kernel_version#v}-${config_version}/lib/modules/${kernel_version#v}"
+	
+	install_cached_component \
+		"kernel" \
+		"${jenkins_url}/job/kata-containers-2.0-kernel-sev-cc-$(uname -m)/${cached_artifacts_path}" \
+		"${kernel_version}" \
+		"$(get_kernel_image_name)" \
+		"${final_tarball_name}" \
+		"${final_tarball_path}" \
+	&& \
+	install_cached_component \
+		"kernel-modules" \
+		"${jenkins_url}/job/kata-containers-2.0-kernel-sev-cc-$(uname -m)/${cached_artifacts_path}" \
+		"${kernel_version}" \
+		"$(get_kernel_image_name)" \
+		"kata-static-cc-sev-kernel-modules.tar.xz" \
+		"${workdir}/kata-static-cc-sev-kernel-modules.tar.xz" \
+	&& \
+	tar xvf "${workdir}/kata-static-cc-sev-kernel-modules.tar.xz" -C  "${module_dir}"\
+	&& return 0
+
+	info "build initramfs for TEE kernel"
+	"${initramfs_builder}"
+	kernel_url="$(yq r $versions_yaml assets.kernel.sev.url)"
+	DESTDIR="${destdir}" PREFIX="${cc_prefix}" "${kernel_builder}" -x "sev" -v "${kernel_version}" -u "${kernel_url}"
+
```

Comment:
The more I look to this code duplication, the more certain I'm this will bite us in the future.

I gave it a try and reworked this bit so we can keep using the same functions as much as possible, and here's the result: https://github.com/fidencio/kata-containers/pull/new/topic/rework-6177

WDYT?